explorer/ast/static_scope.cpp

```diff
@@ -35,20 +35,21 @@ auto StaticScope::Add(std::string_view name, ValueNodeView entity,
   return Success();
 }
 
-void StaticScope::Print(llvm::raw_ostream& out) const {
+template<typename Action>
+void StaticScope::PrintCommon(Action action) const {
```

Comment:
I don't think this will compile -- there's no corresponding member declared in class `StaticScope`. Is it possible that you forgot to commit the changes to `static_scope.h`?

Reply:
That's right -- I've gone back and updated the header file in the PR.